aws_utils/emr/statuschecker.py

```diff
@@ -0,0 +1,93 @@
+import time
+import boto3
+import logging
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class NoSuchActivityError(Exception):
+    """
+    A custom exception
+    """
+
+
+def get_pipeline_state(pipeline_name, region):
+    """
+    Connects to AWS and gets a status of a pipeline
+    :param pipeline_name: (str) name of the Pipeline
+    :param region: (str) AWS region
+    :return: current status of a Pipeline
+    Possible return values : ACTIVATING, CANCELED, CASCADE_FAILED, DEACTIVATING, FAILED, FINISHED, INACTIVE, PAUSED,
+                             PENDING, RUNNING, SHUTTING_DOWN, SKIPPED, TIMEDOUT, VALIDATING, WAITING_FOR_RUNNER, WAITING_ON_DEPENDENCIES
+    """
+    # create a client connection
+    boto3client_pipelines = boto3.client('datapipeline', region) #'us-east-1')
+    pipelines = boto3client_pipelines.list_pipelines()
+
+    current_status_pipeline = ''
+    # get the status of the pipeline
+    for pipeline in pipelines['pipelineIdList']:
+        if pipeline['name'] == pipeline_name:
+            pipeline_settings = boto3client_pipelines.describe_pipelines(pipelineIds=[pipeline['id']])
+            pipeline_fields = pipeline_settings['pipelineDescriptionList'][0]['fields']
+            pipeline_status = [item['stringValue'] for item in pipeline_fields if item['key'] == '@pipelineState']
+            if pipeline_status is not '':
+                return ''.join(pipeline_status)
+            else:
+                raise NoSuchActivityError('\tThe EMR status is empty')
```

Comment:
Do you really need this initial tab in the error description?
Ideally if you need formatting, the code that displays it should do the formatting and not the error itself?